### PR TITLE
Generate the mount_option_var_tmp_bind.xml OVAL check from template

### PR DESCRIPTION
**Pull request by mpreisler**
No need to write it by hand.

I am not 100% sure about the debian and ubuntu paths so I added a fixme.

### CODE REVIEW

File: sle12/templates/csv/mount_options.csv [hunk 1]
```diff
@@ -6,14 +6,4 @@
 #     '$' to reference a variable, e.g. var_removable_partition,nodev)
 #  If the remediation can create (i.e. not just modify) an /etc/fstab line,
 #  add the 'create_fstab_entry_if_needed' literal string as the third argument.
-/dev/shm,nodev
```
Comment (cipherboy) [inline sle12/templates/csv/mount_options.csv]: It is not obvious to me why these are going away in SLE12. Perhaps could split the commit or add a comment on why these are getting removed?

Comment (mpreisler) [reply sle12/templates/csv/mount_options.csv]: I am 99% sure this was an accident, I have updated the PR.

**Comment by cipherboy** [comment]
Thanks @mpreisler; this looks better. I'm not sure I understand the `maybe this should be /tmp` portion (iirc no system has `/tmp` bind-mounted as `tmpfs` is the filesystem backing `/tmp` on modern distros); thoughts?

**Comment by mpreisler** [comment]
> Thanks @mpreisler; this looks better. I'm not sure I understand the maybe this should be /tmp portion (iirc no system has /tmp bind-mounted as tmpfs is the filesystem backing /tmp on modern distros); thoughts?

I think you are correct. I had the comment there because the ubuntu checks were checking for /tmp being bind mounted which I thought was wrong. I have removed that comment because it probably does more harm than good.

**Comment by cipherboy** [comment]
This looks better. Thanks @mpreisler! ACK.

**Pull request merged**